test/message_reader_fuzz_test.cc

```diff
@@ -0,0 +1,45 @@
+#include "grpc_transcoding/message_reader.h"
+
+#include "test_common.h"
+
+#include <fuzzer/FuzzedDataProvider.h>
```

Comment:
This may require clang.   can we just copy the file here?

Reply:
It would be preferable to setup clang, we even mention this in the setup for our repo: https://github.com/grpc-ecosystem/grpc-httpjson-transcoding#toolchain

We should always use clang, Envoy's posture is that it is prefered over gcc. Would you still like me to copy the file over to support both?

Reply:
ok then.  let us just use clang.